### ISSUE TITLE
Queries not running in prombench

### ISSUE THREAD

**Issue by bboreham** [prometheus/test-infra#532]
Pod is crashlooping; in the logs, I see:

```
Scanned up to 27/04/2023, 14:03. Scanned 128.1 KB.
2023-04-27 14:49:37.987 BST
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-04-27 14:49:37.987 BST
TypeError: load() missing 1 required positional argument: 'Loader'
2023-04-27 14:54:45.033 BST
Traceback (most recent call last): File "/usr/src/app/./main.py", line 140, in <module> main() File "/usr/src/app/./main.py", line 121, in main config = yaml.load(open("/etc/loadgen/config.yaml", 'r').read())
2023-04-27 14:54:45.033 BST
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-04-27 14:54:45.033 BST
TypeError: load() missing 1 required positional argument: 'Loader'
2023-04-27 14:59:51.011 BST
Traceback (most recent call last): File "/usr/src/app/./main.py", line 140, in <module> main() File "/usr/src/app/./main.py", line 121, in main config = yaml.load(open("/etc/loadgen/config.yaml", 'r').read())
2023-04-27 14:59:51.011 BST
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by bboreham** [comment]
Could be same issue as #530

**Comment by bboreham** [comment]
Fixed by #531 and #533.
